test/langtools/tools/javac/lambda/IdentifierTest.java

```diff
@@ -5,7 +5,7 @@
  * @summary Test generation of warnings when '_' is used an identifier
  * @compile/fail/ref=IdentifierTest8.out --release 8 -Werror -XDrawDiagnostics -Xlint:-options IdentifierTest.java
  * @compile/fail/ref=IdentifierTest9.out -XDrawDiagnostics IdentifierTest.java
```

Comment:
Note here we are using `ref=IdentifierTest9.out`, but there's no `--release 9` (or `--release 21`). So, it is the same as the next line. I think it would be useful if one of these lines used `--release 21` - even if the outputs are the same.